members/models/address.py

```diff
@@ -21,11 +21,11 @@ class Meta:
     city = models.CharField("By", max_length=200)
     zipcode = models.CharField("Postnummer", max_length=4)
     REGION_CHOICES = (
-        ("Region Syddanmark", "Region Syddanmark"),
-        ("Region Hovedstaden", "Region Hovedstaden"),
-        ("Region Nordjylland", "Region Nordjylland"),
-        ("Region Midtjylland", "Region Midtjylland"),
-        ("Region Sjælland", "Region Sjælland"),
+        ("Region Syddanmark", "Syddanmark"),
+        ("Region Hovedstaden", "Hovedstaden"),
+        ("Region Nordjylland", "Nordjylland"),
+        ("Region Midtjylland", "Midtjylland"),
+        ("Region Sjælland", "Sjælland"),
```

Comment:
Bare slet region fra begge, det kræver så en migration på live.

Reply:
Er der den store årsag til det? For så vidt jeg kan se kræver det ikke bare en migration, men også at vi får ændret hvad der rent faktisk står i databasen, og behandler det vi får fra DAWA

Reply:
Ved at vi ikke bruger det vi selv har fundet på, men det fra DAWA er vi sikre på det altid automatisk kommer på. :) Det var lidt en fejl at lave vores egen inddeling.

Reply:
Er du sikker? For da jeg ændrede det her så gav det mig en "Region Hovedstaden" Key Error, selv efter migration

Reply:
Altså jeg er sikker på at det er begrundelsen for hvorfor vi bør skifte over. :) Men ja det vil kræve lidt arbejde for at gøre det rigtigt. Sig til hvis du har brug for hjælp med det.

Reply:
Tjah, jeg synes det der `region` prefix ikke er så kønt. 
For at ændre det skal du bare køre følgende i en migration (Eller i en shell) .
addresses = Address.objects.all()``

Reply:
Tror bare jeg lader det være foreløbigt så